scripts/db/migrate-db-local.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+SCRIPT_PATH=$( cd $(dirname $0) ; pwd -P )
+MIGRATE_DIRECTORY="$SCRIPT_PATH/../../postgres/migrate-db"
+
+echo "change directory to $DATA_DIRECTORY"
+
+cd $MIGRATE_DIRECTORY
+
+source ~/.nvm/nvm.sh
+nvm use
```

Comment:
as above - maybe let's just leave it up to the developer to set the right node version